docs/editors/sublime.md

```diff
@@ -76,42 +64,12 @@ in insert-mode.
 
 ## Goto symbol in workspace
 
-You can search for symbols in your dependency source using the command palette   
+You can search for symbols in your dependency source using the command palette
 
 ![workspace symbols](https://i.imgur.com/8X0XNi2.gif)
 
 ## Manually trigger build import
 
-You can configure a custom command "Metals: Import Build" to manually trigger
```

Comment:
I'm assuming that this is no longer needed?

Reply:
Yes it is part of the package commands [here](https://github.com/scalameta/metals-sublime/blob/master/metals-sublime.sublime-commands)

Reply:
Cool, that's what I figured. 👍